packages/pyright-internal/src/analyzer/service.ts

```diff
@@ -29,16 +29,16 @@ import { BackgroundAnalysisBase, IndexOptions, RefreshOptions } from '../backgro
 import { CancellationProvider, DefaultCancellationProvider } from '../common/cancellationUtils';
 import { CommandLineOptions } from '../common/commandLineOptions';
 import { ConfigOptions, matchFileSpecs } from '../common/configOptions';
-import { ConsoleInterface, log, LogLevel, StandardConsole } from '../common/console';
+import { ConsoleInterface, LogLevel, StandardConsole, log } from '../common/console';
```

Comment:
I am not sure why these ordering is changed. pyright and pylance might have different prettier setting?